scripts/player.gd

```diff
@@ -43,3 +52,23 @@ func set_coyote_jump_enabled(value: bool) -> void:
 
 func is_coyote_jump_enabled() -> bool:
 	return !$CoyoteJumpTimer.is_stopped()
+
+
+## Returns the amount of water in the meter.
```

Comment:
Conisder putting the water level bar in a seperate script attached to the corresponding water bar node?
Just so that the player script is more modular and specific